legal-api/src/legal_api/services/authz.py

```diff
@@ -333,14 +333,14 @@ def get_allowable_filings_dict():
                     'fullRestoration': {
                         'legalTypes': ['BC', 'BEN', 'CC', 'ULC', 'C', 'CBEN', 'CUL', 'CCC'],
                         'blockerChecks': {
-                            'invalidStateFilings': ['continuationIn', 'continuationOut'],
```

Comment:
`continuationIn` shouldn't be here

Reply:
Just so I'm clear, this is to prevent a full restoration if the last filing in the ledger is a continuation in application, which can't happen if this business needs a full restoration (ie, is historical), right?

Does a continuation in application count as a state filing anyway?

Reply:
Right. We don't count `continuationIn` as a state filing